snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SfvChecksumImpl.java

```diff
@@ -122,6 +122,15 @@ public void updateFromFile(final Path filePath) throws IOException {
     checksums.put(fileName, checksum.getValue());
   }
 
+  @Override
+  public void updateFromBytes(final String fileName, final byte[] bytes) {
+    combinedChecksum.update(fileName.getBytes(UTF_8));
+    final Checksum checksum = new CRC32C();
```

Comment:
:wrench: You can create it once for the whole class, and just call `CRC32C#reset` before every usage. Very minor optimization :upside_down_face:

Reply:
I think I keep it for now.

The CRC323C#reset :D 

```
    public void reset() {
        this.crc = -1;
    }
```

Reply:
If you look at how it works, this is the initial value :upside_down_face: Then when you pass things it gets updated. So it is, in fact, resetting the state to the initial values ;)